autopark/VehicleType.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace autopark
+{
+    class VehicleType
+    {
+        string typeName;
+        float taxCoefficient; //or float taxCoefficient { get; set; }
+
+        public float TaxCoefficient
+        {
+            get { return taxCoefficient; }
+            set { taxCoefficient = value; }
+        }
```

Comment:
Use Auto Property, if you don't need special logic for field. Auto Properties makes code more readable and compacted.

Reply:
![image](https://user-images.githubusercontent.com/79978383/115709790-4beb9500-a37a-11eb-964b-5f8fcc5aa593.png)
Done